contingency/contingency-api/src/main/java/com/powsybl/contingency/contingency/list/IdentifierContingencyList.java

```diff
@@ -68,4 +67,22 @@ public List<Contingency> getContingencies(Network network) {
                 .filter(contingency -> contingency.isValid(network))
                 .collect(Collectors.toList());
     }
+
+    public Map<String, Set<String>> getNotFoundElements(Network network) {
+        Map<String, Set<String>> notFoundElementsMap = new HashMap<>();
+        networkElementIdentifiers.forEach(identifier -> {
+            Set<String> notFoundElements = identifier.getNotFoundElements(network);
+            if (!notFoundElements.isEmpty()) {
+                String contingencyId = identifier.getContingencyId().orElse("Contingency : " +
```

Comment:
You can share the contingency id creation with the getContingencies method

Reply:
done